packages/terra-notification-dialog/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Unreleased
 3.9.0 - (August 14, 2019)
 ------------------
 ### Changed
+* Update wdio snapshots with new button focus outline styles
```

Comment:
This bullet entry is correct, I updated the changelog committed that to master and then saw the wdio test failure afterwards. Plan is to get this out today with this release.